styles/_theme.scss

```diff
@@ -1,3 +1,93 @@
+// Neutral colors
+$palette-neutral-100: var(--fw-palette-neutral-100, $color-milk);
+
+// Primary colors
+$palette-primary-900: var(--fw-palette-primary-900, $color-elephant-900);
+$palette-primary-800: var(--fw-palette-primary-800, $color-elephant-800);
+$palette-primary-700: var(--fw-palette-primary-700, $color-elephant-700);
+$palette-primary-600: var(--fw-palette-primary-50, $color-elephant-600);
+
+// Secondary colors
+$palette-secondary-700: var(--fw-palette-secondary-700, $color-smoke-700);
+$palette-secondary-400: var(--fw-palette-secondary-300, $color-smoke-400);
```

Comment:
Please change it to `--fw-palette-secondary-400`

Reply:
Updated the css variable